pkg/controller/node/ipam/cloud_cidr_allocator.go

```diff
@@ -49,9 +49,10 @@ type cloudCIDRAllocator struct {
 	client clientset.Interface
 	cloud  *gce.GCECloud
 
-	// Channel that is used to pass updating Nodes with assigned CIDRs to the background
-	// This increases a throughput of CIDR assignment by not blocking on long operations.
-	nodeCIDRUpdateChannel chan nodeAndCIDR
+	// Channel that is used to pass updating Nodes to the background.
+	// This increases a throughput of CIDR assignment by not blocking
+	// on long operations (which can't be done from event handlers).
+	nodeUpdateChannel chan string
 	recorder              record.EventRecorder
```

Comment:
This might cause gofmt to fail because of misaligned spaces.

Reply:
done